src/app/remotes/announcement-banner/announcement-banner.component.html

```diff
@@ -0,0 +1,45 @@
+<ng-container *ngIf="announcements$ | async as announcements">
+  <p-carousel
+    [value]="announcements"
+    [numVisible]="1"
+    [numScroll]="1"
+    *ngIf="announcements && announcements.length > 0"
+    [showIndicators]="false"
```

Comment:
`showIndicators` was set to false to prevent the announcement-banner component from being too big. Additionally the indicators might look weird when combined with some workspace themes. See the attached screenshots for a comparison.

Without Indicators
![grafik](https://github.com/onecx/onecx-announcement-ui/assets/55296998/0983c750-916c-45d9-b4b0-1055376244f0)

With Indicators
![grafik](https://github.com/onecx/onecx-announcement-ui/assets/55296998/999eb6d5-d75c-4786-a2e8-c8fe4ba84719)